### PR TITLE
Blog - chown: Changing File and Directory File in Linux

### PR DESCRIPTION
**Pull request by msali123**
done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac [Preview](https://645a78a19bb5210581960f4c--ephemeral-narwhal-3ac3b2.netlify.app/blog/chown-permissions-chmod/)

File: blog/_posts/2023-06-15-chown-permissions-chmod.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+internal-links:
+ - Linux
+ - Directory Ownership
+ - File Ownership
+ - chown
+ - Permissions
+---
+
+**We're [Earthly](https://earthly.dev/). We make building software simpler and, therefore, faster. This article is about Linux file permissions and ownership If you enjoy doing things at command line you might like [Earthly](https://earthly.dev/). It's a pretty great open-source build tool.**
```
Comment (balapriyac) [inline blog/_posts/2023-06-15-chown-permissions-chmod.md:16]:
```suggestion
**We're [Earthly](https://earthly.dev/). We make building software simpler and, therefore, faster. This article is about Linux file permissions and ownership. If you enjoy doing things at the command line you might like [Earthly](https://earthly.dev/). It's a pretty great open-source build tool.**
```

File: blog/_posts/2023-06-15-chown-permissions-chmod.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+Try the `chown` command as the root user:
+
+~~~{.bash caption=">_"}
+demo@localhost:~$ sudo -i
+root@localhost:~# chown notdemo /home/demo/todo.txt
+root@localhost:~# exit
+~~~
+
+> **Please note:** Your Linux configuration or distribution may have a different method for interacting with or becoming the `root` user. Consult with your specific online documentation on how to use the [`sudo`](https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/c6239.htm) command properly.
```
Comment (balapriyac) [inline blog/_posts/2023-06-15-chown-permissions-chmod.md:105]:
```suggestion
<div class="notice--info">
Your Linux configuration or distribution may have a different method for interacting with or becoming the `root` user. Consult with your specific online documentation on how to use the [`sudo`](https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/c6239.htm) command properly.
</div>
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
@msali123 Nice work! Suggested a couple of formatting edits.

**Comment by msali123** [comment]
Thanks Bala [changes done](https://64633a63c756155f9b4663af--ephemeral-narwhal-3ac3b2.netlify.app/blog/chown-permissions-chmod/)

**Comment by balapriyac** [comment]
@jalletto this is ready to merge.

**Review by jalletto — APPROVED** [review]

**Pull request merged**